src/mono/wasm/Wasm.Build.Tests/Templates/WasmTemplateTests.cs

```diff
@@ -407,23 +407,21 @@ public void ConsolePublishAndRun(string config, bool aot, bool relinking)
         }
 
         [Theory]
-        [InlineData("", BuildTestBase.DefaultTargetFramework, DefaultRuntimeAssetsRelativePath)]
-        [InlineData("", BuildTestBase.DefaultTargetFramework, "./")]
```

Comment:
Why are we removing this? Is `WasmRuntimeAssetsLocation` not supported now?

Reply:
Disable those unsupported cases then, instead of removing them. And reference a relevant issue.